UNRELEASED.md

```diff
@@ -2,6 +2,15 @@
 
 > Place your changes below this line.
 
+**Breaking:**
+  - `BpkRating`
+    - `ratingOrientation` XML attribute has been renamed to `ratingOrientation`
```

Comment:
Should this upper line be here? It says it's been renamed to the same thing

Reply:
Nice catch! Should be fixed now.